src/twisted/__init__.py

```diff
@@ -12,7 +12,7 @@
 
 __version__ = version.short()
 
-if sys.version_info < (3, 5):
+if sys.version_info < (3, 6):
     raise Exception(
-        "This version of Twisted is not compatible with Python 3.4 " "or below."
+        "This version of Twisted is not compatible with Python 3.5 or below."
     )
```

Comment:
I propose that we remove this check.  It's uncovered, so blocking this PR, and `setup.py` does the job already.